client/src/components/Header/index.tsx

```diff
@@ -85,7 +85,9 @@ export class Header extends React.Component<
       <>
         <Helmet>
           <style>
-            {':root{--header-height: 38px; --lang-menu-height: 22.5rem;}'}
+            {
+              ':root{--header-height: 38px; --search-box-form: 38; --lang-menu-height: 22.5rem;}'
```

Comment:
```suggestion
              ':root{--header-height: 38px; --search-box-form: 38px; --lang-menu-height: 22.5rem;}'
```
Shouldn't it still be px?